internal/operators/odf/validations.go

```diff
@@ -101,7 +100,7 @@ func (o *operator) computeResourcesAllNodes(cluster *models.Cluster, odfClusterR
 	return status, nil
 }
 
-func (o *operator) computeNodeResourceUtil(host *models.Host, odfClusterResources *odfClusterResourcesInfo, mode odfDeploymentMode) (string, error) {
+func (o *operator) computeNodeResourceUtil(cluster *models.Cluster, host *models.Host, odfClusterResources *odfClusterResourcesInfo, mode odfDeploymentMode) (string, error) {
```

Comment:
Is `mode` no longer being used in this function?

Reply:
Mode was used just for the error message for ODF, so it's now handled in a wrapper for odf only.

Reply:
Are we able to remove it from this function then?

Reply:
oops sorry, I've misread where you where commenting, I was referring to another function where mode was use.
You're right, is not used anymore, we can remove it and simplify this function